docs/BASIC_SETUP.md

```diff
@@ -4,22 +4,20 @@
 
 mLRS can work with any radio which provides an SBus output, which should be really every radio. This page describes this most basic setup.
```

Comment:
Perhaps this could be 'This page describes a basic setup using a radio which provides an SBus output.'

Reply:
FANTASTIC! done

Reply:
Think that the first sentence could be removed entirely.

Reply:
it is true thst the loss of technically relevant info is zero with removing it. I would miss the advertising aspect of it however LOL. But the other pages don't have that either ... so, probably you are correct :D

Maybe one can carry over the Notes from the CRSF page (with adaption). What do you think?

Reply:
and it's INAV and not iNav LOL...

Reply:
> Maybe one can carry over the Notes from the CRSF page (with adaption). What do you think?

Yes, this makes sense to me.

Reply:
done